src/vite.config.js

```diff
@@ -0,0 +1,16 @@
+const { resolve } = require('path')
+
+module.exports = {
+  build: {
+    rollupOptions: {
+      input: {
+        main: resolve(__dirname, 'index.html'),
+        buddies: resolve(__dirname, 'code-buddys.html'),
+        teams: resolve(__dirname, 'teams.html'),
+        energy: resolve(__dirname, 'energy.html'),
+        journal: resolve(__dirname, 'journal.html'),
+        journalForm: resolve(__dirname, 'journal-form.html'),
+      },
+    },
+  },
+}
```

Comment:
These additions in the config tell the build script to include other files, besides the `index.html`, as well when creating a production build